phalcon/Storage/Adapter/AbstractAdapter.zep

```diff
@@ -203,11 +203,8 @@ abstract class AbstractAdapter implements AdapterInterface
     /**
      * Returns unserialized data
      */
-    protected function getUnserializedData(var content, var defaultValue = null) -> var
+    protected function getUnserializedData(var content) -> var
```

Comment:
What about to make `content` as string type?

Reply:
if you have saved just `false` this will troll a warning i think https://github.com/phalcon/cphalcon/blob/5.0.x/phalcon/Storage/Adapter/Redis.zep#L110

Reply:
i will need some time to try out if string will do the trick, cuz if feels wrong

Reply:
yes, so... if you are not using a serializer, the logic breaks hard cuz

```php
[TypeError] Phalcon\Storage\Adapter\AbstractAdapter::getUnserializedData(): Argument #1 ($content) must be of type string, array given

[TypeError] Phalcon\Storage\Adapter\AbstractAdapter::getUnserializedData(): Argument #1 ($content) must be of type string, stdClass given
```